Aztec/Classes/Private/Libxml2/Data/ElementNode.swift

```diff
@@ -52,7 +54,7 @@ extension Libxml2 {
             /// Returns an array with all block-level elements.
             ///
             static func blockLevelNodeNames() -> [StandardName] {
-                return [.Address, .Blockquote, .Div, .Dl, .Fieldset, .Form, .H1, .H2, .H3, .H4, .H5, .H6, .Hr, .Noscript, .Ol, .P, .Pre, .Table, .Ul]
+                return [.Address, .Blockquote, .Div, .Dl, .Fieldset, .Form, .H1, .H2, .H3, .H4, .H5, .H6, .Hr, .Noscript, .Ol, .P, .Pre, .Table, .Ul, .Img]
```

Comment:
Why did you add `.Img` here?  It's not a block-level node... maybe we need to change the logic of a method instead?

Let me know if I can help with this: but the bottom line is we must avoid tagging `.Img` as a block-level node.